modules/userId/index.js

```diff
@@ -77,6 +78,7 @@ import CONSTANTS from '../../src/constants.json';
 import {module} from '../../src/hook';
 import {unifiedIdSubmodule} from './unifiedIdSystem.js';
 import {pubCommonIdSubmodule} from './pubCommonIdSystem.js';
+import {identityLinkSubmodule} from './identityLinkSystem.js';
```

Comment:
Remove the import for `identityLinkSubmodule` - the identity link submodule handles the registering in it’s submodule js.

Reply:
Done